docs/src/topics/firewalling.md

```diff
@@ -73,3 +79,31 @@ spec:
         - ports:
             - port: "22"
 ```
+
+## Cloud Firewalls
+
+For controlling firewalls via Linode resources, a [Cloud Firewall](https://www.linode.com/products/cloud-firewall/) can
+be defined and provisioned via `LinodeFirewall` resources in CAPL. The created Cloud Firewall's ID can then be used in
+a `LinodeMachine` or a `LinodeMachineTemplate`'s `firewallID` field. Note that the `firewallID` field is currently
+immutable, so it must be set at creation time).
+
+Example `LinodeFirewall`:
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: LinodeFirewall
+metadata:
+  name: sample-fw
+spec:
+  enabled: true
+  inboundPolicy: DROP
+  inboundRules:
+    - action: ACCEPT
+      label: k8s-api
+      ports: "6443"
+      protocol: "TCP"
+      addresses:
+        ipv4:
+          - "172.232.1.1/16"
+```
```

Comment:
I think it would be nice to have outboundrules included in the example!

Reply:
Should be the same configuration as the inboundRules